config.yaml

```diff
@@ -9,3 +9,37 @@ options:
       take place whenever an update is available. For example, 03-05 will allow updates to take
       place from 3AM UTC to 5AM UTC. Awaits for running job completion for 5 minutes. 
     default: ""
+  proxy-hostname:
```

Comment:
Shouldn't be fetching these from the juju model config?

Reply:
Sorry if I didn't understand correctly, would you mean that this should be fetched from the juju model config from default?

Reply:
Yes this should be done via model configurations, @weiiwang01 has a PR open for flask to do something similar

Reply:
https://github.com/canonical/flask-k8s-operator/pull/33